editor/import/resource_importer_texture.cpp

```diff
@@ -38,6 +38,7 @@
 #include "editor/editor_node.h"
 #include "editor/editor_scale.h"
 #include "editor/editor_settings.h"
+#include "resource_importer_texture_settings.h"
```

Comment:
Use absolute paths for editor includes (`editor/import/resource_importer_texture-settings.h`).